src/main/java/org/example/model/Cat.java

```diff
@@ -0,0 +1,68 @@
+package org.example.model;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cat {
+
+    private int id;
+
+    private String name;
+
+    private int age;
+
+    private String breed;
+
+    private List<Owner> owners = new ArrayList<>();
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public int getAge() {
+        return age;
+    }
+
+    public void setAge(int age) {
+        this.age = age;
+    }
+
+    public String getBreed() {
+        return breed;
+    }
+
+    public void setBreed(String breed) {
+        this.breed = breed;
+    }
+
+    public List<Owner> getOwners() {
+        return owners;
+    }
+
+    public void addOwner(Owner owner) {
+        owners.add(owner);
+    }
+
+    @Override
+    public String toString() {
+        return "Cat{" +
```

Comment:
You may have done this elsewhere, but consider warning students about generated toStrings.
Like if this Cat class's toString printed out the toStrings of the Owners and Owner printed out the toStrings of Cats.
I think I remember some of the BlackRock students dealing with circular toString issues.

Reply:
I added a warning about this in the lab, I'll update the lesson too.